src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -57,7 +57,7 @@ public HttpResponseMessage CreateCheckStatusResponse(HttpRequestMessage request,
         /// <exception cref="ArgumentException">
         /// The specified function does not exist, is disabled, or is not an orchestrator function.
         /// </exception>
-        public Task<string> StartNewAsync(string orchestratorFunctionName, object input)
+        public virtual Task<string> StartNewAsync(string orchestratorFunctionName, object input)
```

Comment:
Since this overload just calls the other `StartNewAsync` overload, let's move it into the base class and make it virtual. The more complex overload can be made abstract.